### PR TITLE
Testing testing one two

### PR DESCRIPTION
**Pull request by spike01**

### CODE REVIEW

File: src/wordCounter.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+//For future implementation
```
Comment (leoallen85) [inline src/wordCounter.js:1]: commented code!

Comment (spike01) [inline src/wordCounter.js:1]: No Hound errors? Amazing.

Comment (spike01) [inline src/wordCounter.js:1]: It's almost like I wrote this app

**Comment by spike01** [comment]
Any idea why Travis isn't trying to rebuild?

**Pull request closed without merging**